.github/workflows/qa.yaml

```diff
@@ -90,17 +90,33 @@ jobs:
         run: |
           set -eu
           cargo install grcov
-      - name: Run tests
+      - name: Run tests (with coverage collection)
```

Comment:
Since the command got way more complex than before, I thought the step could use an update in its naming...